src/main/resources/lib/nextjs/proxy.es6

```diff
@@ -326,7 +317,7 @@ function initNextjsCookieName(request, site) {
         paramString = '';
     }
     // create separate data for different params too
-    COOKIE_DATA_KEY = `NEXTJS_DATA_FOR_${request.mode}_AT_${site._name}_WITH_${paramString}`;
+    COOKIE_DATA_KEY = `NEXTJS_DATA_BY_${getJSessionId(request)}_FOR_${request.mode}_AT_${site._name}_WITH_${paramString}`;
```

Comment:
Since this is included on every request should we not try to keep it as short as possible?